recipes/cassiopee/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: cassiopee
+  version: "1.0.5"
+
+build:
+  number: 0
+
+source:
+  fn: 1.0.5.tar.gz
+  url: https://github.com/osallou/cassiopee-c/archive/1.0.5.tar.gz
+  md5: 38f07e5d2ee2a55679281bce73f9993e
+
+requirements:
+  build:
+    - glog
+    - cppunit
+    - boost =={{CONDA_BOOST}}
+    - autoconf
+    - gcc   # [not osx]
+    - llvm  # [osx]
+    - cmake
+    - icu 58.*
+    - bzip2
+    - zlib
```

Comment:
zlib needs to be pinned with `{{CONDA_ZLIB}}`

Reply:
done